tests/test_init.py

```diff
@@ -19,90 +19,241 @@
-                      (qml.init.interferometer_phi_uniform, base_uni, (n_if,)),
-                      (qml.init.interferometer_varphi_uniform, base_uni, (n_wires,)),
-                      (qml.init.qaoa_embedding_normal, rpt_nrml, (repeat, 2*n_wires)),
-                      (qml.init.qaoa_embedding_uniform, rpt_uni, (repeat, 2*n_wires)),
-                      (qml.init.qaoa_embedding_uniform, {'n_layers': 2, 'n_wires': 1, 'low': 0, 'high': 1}, (2, 1)),
-                      (qml.init.qaoa_embedding_uniform, {'n_layers': 2, 'n_wires': 2, 'low': 0, 'high': 1}, (2, 3)),
-                      (qml.init.qaoa_embedding_normal, {'n_layers': 2, 'n_wires': 1, 'mean': 0, 'std': 1}, (2, 1)),
-                      (qml.init.qaoa_embedding_normal, {'n_layers': 2, 'n_wires': 2, 'mean': 0, 'std': 1}, (2, 3)),
-                      (qml.init.simplified_two_design_initial_layer_uniform, {'n_wires': 1, 'low': 0, 'high': 1}, (1,)),
-                      (qml.init.simplified_two_design_initial_layer_uniform, {'n_wires': 3, 'low': 0, 'high': 1}, (3,)),
-                      (qml.init.simplified_two_design_initial_layer_normal, {'n_wires': 1, 'mean': 0, 'std': 1}, (1,)),
-                      (qml.init.simplified_two_design_initial_layer_normal, {'n_wires': 3, 'mean': 0, 'std': 1}, (3,)),
-                      (qml.init.simplified_two_design_weights_uniform, {'n_layers': 2, 'n_wires': 1, 'low': 0, 'high': 1}, (0,)),
-                      (qml.init.simplified_two_design_weights_uniform, {'n_layers': 2, 'n_wires': 2, 'low': 0, 'high': 1}, (2, 1, 2)),
-                      (qml.init.simplified_two_design_weights_uniform, {'n_layers': 2, 'n_wires': 4, 'low': 0, 'high': 1}, (2, 3, 2)),
-                      (qml.init.simplified_two_design_weights_normal, {'n_layers': 2, 'n_wires': 1, 'mean': 0, 'std': 1}, (0, )),
-                      (qml.init.simplified_two_design_weights_normal, {'n_layers': 2, 'n_wires': 2, 'mean': 0, 'std': 1}, (2, 1, 2)),
-                      (qml.init.simplified_two_design_weights_normal, {'n_layers': 2, 'n_wires': 4, 'mean': 0, 'std': 1}, (2, 3, 2)),
-                      (qml.init.basic_entangler_layers_normal, {'n_layers': 2, 'n_wires': 1, 'mean': 0, 'std': 1},
+INIT_KWARGS_SHAPES = [(qml.init.random_layers_normal,
+                       {'n_layers': 2, 'n_wires': 3, 'n_rots': 10, 'mean': 0, 'std': 1},
+                       (2, 10)),
+                      (qml.init.random_layers_normal,
+                       {'n_layers': 2, 'n_wires': 1, 'n_rots': 10, 'mean': 0, 'std': 1},
+                       (2, 10)),
+                      (qml.init.random_layers_normal,
+                       {'n_layers': 2, 'n_wires': 3, 'n_rots': None, 'mean': 0, 'std': 1},
+                       (2, 3)),
+                      (qml.init.strong_ent_layers_normal,
+                       {'n_layers': 2, 'n_wires': 3, 'mean': 0, 'std': 1},
+                       (2, 3, 3)),
+                      (qml.init.strong_ent_layers_normal,
```

Comment:
Out of scope, just for my own understanding: what is the difference between:
* `cvqnn_layers_theta_uniform`
* `cvqnn_layers_phi_uniform`
* `cvqnn_layers_varphi_uniform`?

Are they separated such upon seeding one gets different results?

Reply:
None, only so the user has one for every parameter of the template, without having to know that they are the same.

Reply:
I see! Could it make sense to merge them into one instead and the user could e.g. pass a `parameter` argument for the name?

Reply:
Yes, in future this will be the way!